src/kibana/components/vislib/lib/data.js

```diff
@@ -244,6 +244,22 @@ define(function (require) {
     };
 
     /**
+     * get min and max for all cols, rows of data
+     *
+     * @method getMaxMin
+     * @return {Object}
+     */
+    Data.prototype.getGeoExtents = function () {
+      var visData = this.getVisData();
+
+      return _.reduce(_.pluck(visData, 'geoJson.properties'), function (minMax, props) {
+        if (minMax.min > props.min) minMax.min = props.min;
+        if (minMax.max < props.max) minMax.max = props.max;
+        return minMax;
```

Comment:
It probably makes sense to use `Math.min/max` here... something like `minMax.min = Math.min(minMax.min, props.min);`